app/views/content/life-as-a-teacher/teaching-as-a-career/what-its-like-to-be-a-teacher/_article.html.erb

```diff
@@ -40,7 +40,7 @@
  <p> When you're training, you'll work with a teacher who will help you plan and may share some of their own planning with you. Over time, with the help of other colleagues, you'll get a better feel for what makes a good lesson and build a bank of resources you can reuse to cut down on your workload.</p>
    <p>Teachers are entitled to 10% of their timetable to do planning, preparation, and assessment (PPA). Depending on the school, this could mean that you have a couple of free periods next to each other, or they could be spread across the week.</p>
  <%= render 'content/shared/quotes/quote_food_helen' %>
-<p>To find out more about what it's like to be a teacher, <a href="/life-as-a-teacher/explore-subjects">explore subjects you could teach</a>.</p>
+<p>If teaching sounds like the career for you, find out more about <a href="/steps-to-become-a-teacher">how to become a teacher</a>.</p>
```

Comment:
Does this need a heading above it such as `Next steps` (or something better!)? It doesn't really fit under `Workload`?